datalad_dataset_addition_procedure.md

```diff
@@ -1,6 +1,22 @@
-# Datalad dataset addition procedure
+# Datalad dataset addition procedures
 
-This procedure is intended for developers.
+This documentation is intended for developers.
+
+# Requirements
+
+All datasets must include a `README.md` in the root directory.
+Adding meta-data about your dataset is required. 
+
+All datasets must include a `DATS.json` metadata file in the root directory as described in [the main documentation page](https://github.com/CONP-PCNO/conp-documentation/blob/master/CONP_main_data_documentation.htm).
```

Comment:
This is not required when crawling from Zenodo, as the crawler would create a README and DATS file if they don't exist. I'd move that to the DataLad section.